spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/ServiceCombRBACProperties.java

```diff
@@ -42,6 +41,11 @@
 
   private String password;
 
+  @Autowired(required = false)
+  public void setCiphers(List<Cipher> ciphers) {
```

Comment:
`@Autowired(required = false)  `

if `required = false` can be removed ? setter injection default to not required.